src/modules/AutoinstSoftware.rb

```diff
@@ -826,11 +827,16 @@ def Write
       ok = true
 
       Packages.Init(true)
+      selected_base_products = Product.FindBaseProducts.map { |p| p["name"] }
       # Resetting package selection of previous runs. This is needed
       # because it could be that additional repositories 
       # are available meanwhile. (bnc#979691)
       Pkg.PkgApplReset
```

Comment:
Packages.Reset([:product]) is not working because the product has been set with the level "application high". This call ignores "selected by the user" only.

Reply:
Hm, it should just ignore the products selected by solver (https://github.com/yast/yast-packager/blob/SLE-12-SP5/src/modules/Packages.rb#L815).

But if this works for you I'm fine with that.